pkg/mimirpb/mimir.proto

```diff
@@ -34,6 +34,18 @@ message WriteRequest {
 
 message WriteResponse {}
 
+enum ErrorCause {
+  REPLICAS_DID_NOT_MATCH = 0;
```

Comment:
Can we add an `INVALID = 0` const here? That would prevent the code from silently being interpreted as `REPLICAS_DID_NOT_MATCH` if there's a bug somewhere and cause isn't set.

Reply:
Sure, I can do it.